sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Description>This is the implementation of the Azure Client Pipeline</Description>
     <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
-    <Version>1.37.0-beta.1</Version>
+    <Version>1.37.0-beta.2</Version>
```

Comment:
We don't ship betas of Azure.Core.  Please add any beta features to Azure.Core.Experimental first.

Reply:
OK. Thanks.

Reply:
> Yes. I will put all Azure.Core library files in Azure.Core.Experimental first.

Great!  You should also be able to put any System.ClientModel files in Azure.Core.Experimental as well in order to beta them before shipping them GA in one of the GA Core libraries.